src/sdk/bucketspace_fs.js

```diff
@@ -85,6 +85,14 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             if (account.nsfs_account_config.distinguished_name) {
                 account.nsfs_account_config.distinguished_name = new SensitiveString(account.nsfs_account_config.distinguished_name);
             }
+            try {
```

Comment:
@guymguym I think we need to reconsider having read_account_by_access_key and other new account related functions in bucketspace_fs, it worked before when we didn't have accountspaceFS but now I think it just weird

Reply:
I think that Guy would agree with you from what I understand while I worked on IAM).
I will mention this as a GAP in the PR description.